dao/src/main/java/org/thingsboard/server/dao/sql/query/EntityDataAdapter.java

```diff
@@ -86,8 +87,8 @@ static String convertValue(Object value) {
                     return strVal;
                 }
                 try {
-                    long longVal = Long.parseLong(strVal);
-                    return Long.toString(longVal);
+                    BigInteger longVal = new BigInteger(strVal);
+                    return longVal.toString();
                 } catch (NumberFormatException ignored) {
```

Comment:
Please add tests for the case when NumberFormatException is thrown here, and double conversion happens.